test/api/junction_detection_test.cpp

```diff
@@ -120,6 +125,65 @@ TEST(junction_detection, method_1_and_2)
                                        tmp_dir/"detect_breakends_out_short.fasta",
                                        {1, 2},
                                        simple_clustering,
+                                       no_refinement,
+                                       sv_default_length);
+
+    std::string std_cout = testing::internal::GetCapturedStdout();
+    seqan3::debug_stream << "std_out:\n" << std_cout << '\n';
+    EXPECT_EQ(expected, std_cout);
+
+    // cleanup
+    std::filesystem::remove(tmp_dir/"detect_breakends_out_short.fasta");        // remove output
+}
+
+/* -------- refinement methods tests -------- */
+
+TEST(junction_detection, refinement_method_sViper)
+{
+    std::string expected{
+        "Reference\tchr22\t17458417\tForward\tReference\tchr21\t41972615\tForward\t1\n"
+        "Reference\tchr22\t17458418\tForward\tReference\tchr21\t41972616\tForward\t2\n"
+        "Reference\tm2257/8161/CCS\t41972616\tForward\tRead\t0\t2294\tForward\t1\n"
+        "Reference\tm2257/8161/CCS\t41972616\tReverse\tRead\t0\t3975\tReverse\t1\n"
+    };
+
+    std::filesystem::path tmp_dir = std::filesystem::temp_directory_path();     // get the temp directory
+    std::filesystem::remove(tmp_dir/"detect_breakends_out_short.fasta");        // remove old output if existent
+
+    testing::internal::CaptureStdout();
+    detect_junctions_in_alignment_file(DATADIR"simulated.minimap2.hg19.coordsorted_cutoff.sam",
+                                       tmp_dir/"detect_breakends_out_short.fasta",
+                                       {1, 2, 3, 4},
+                                       simple_clustering,
+                                       sViper_refinement_method,
+                                       sv_default_length);
+
+    std::string std_cout = testing::internal::GetCapturedStdout();
+    seqan3::debug_stream << "std_out:\n" << std_cout << '\n';
```

Comment:
Is there a reason you decided to print the `std::out` output, but not actually test that it gives what's expected? Just wondering. I would say we should either test the output, or just remove the printing.

Reply:
Doesn't the line after that test just that?